descriptions/api.github.com/api.github.com.yaml

```
@@ -9266,6 +9266,11 @@ paths:
                   description: Either `true` to allow rebase-merging pull requests,
                     or `false` to prevent rebase-merging.
                   default: true
+                allow_auto_merge:
+                  type: boolean
+                  description: Either `true` to allow auto-merge on pull requests,
+                    or `false to disallow auto-merge.
```

There is a typo here and on line 12694 @bruce @mattpage 

```suggestion
                    or `false` to disallow auto-merge.
```